README.md

```diff
@@ -77,3 +77,4 @@ spec:
 ## Resources
 
 - [Advanced topics](https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/) (generic / operator framework)
+- [API Reference](https://github.com/netobserv/network-observability-operator/docs/crd.md)
```

Comment:
I don't think this link will work, it would be something like `.../blob/main/docs/crd.md` instead
But even better would be to make a relative link, so that it's consistent across branches
I think `[API Reference](./docs/crd.md)` would work

Reply:
Also can you rename `crd.md` to `FlowCollector.md` ? as there could be more than one CRD

Reply:
sure, here we go